### PR TITLE
Rename

### PR DESCRIPTION
**Pull request by troydai**
@NTaylorMullen any of these variables in build scripts are used on CI?

### CODE REVIEW

**Comment by troydai** [comment]
@NTaylorMullen @anurse 

The dnvm included is copied from https://github.com/aspnet/dotnetsdk/tree/anurse/super-secret-open-source-magic-stuff. I'll keep it updated as it is being reviewed.

**Comment by NTaylorMullen** [comment]
@troydai several, when this gets in we'll also need to run across all the CI configurations.. :shipit:

**Comment by troydai** [comment]
Yup, can't verify it now, pending on kvm => dnvm and klr changes.

**Comment by anurse** [comment]
looks good to me

**Comment by troydai** [comment]
@NTaylorMullen ok if i get this in?

**Comment by muratg** [comment]
:shipit:

File: build/_k-restore.shade [hunk 1]
```diff
@@ -4,7 +4,7 @@ k-restore
     Restores nuget packages required for k projects. Downloads and executes k sdk tools.
 */}
 
-default restore_options='${E("KPM_restore_options")}'
+default restore_options='${E("NUGET3_restore_options")}'
```
Comment (Eilon) [inline build/_k-restore.shade]: Hmm not sure we wanted to do the kpm -> nuget3 rename, did we? Not quite sure where that lands yet... @davidfowl ?

File: build/_k-restore.shade [hunk 2]
```diff
@@ -4,7 +4,7 @@ k-restore
     Restores nuget packages required for k projects. Downloads and executes k sdk tools.
```
Comment (Eilon) [inline build/_k-restore.shade]: Lots of other "k" stuff in here (including filenames) - do we have a work item for these?

Comment (muratg) [inline build/_k-restore.shade]: These should be changed when we remove k.cmd and friends. There's a work item in the other repo.

File: build/_k-restore.shade (hunk 1 above)
Comment (troydai) [inline build/_k-restore.shade]: The name is entirely used internally. Not consumed by customers. They can be renamed anytime.

**Pull request merged**